packages/dds/tree/src/shared-tree/schematizingTreeView.ts

```diff
@@ -47,6 +47,7 @@ export class SchematizingSimpleTreeView<in out TRootSchema extends ImplicitField
 
 	private readonly viewSchema: ViewSchema;
 
+	private readonly unregisterCallbacks = new Set<() => void>();
```

Comment:
why a set? aren't the callbacks object identity and thus couldn't be duplicated?

Reply:
oh, for individual delete. nvm.